supporting_content/dimensions/commitmentdiscounttype.md

```diff
@@ -0,0 +1,55 @@
+# Column: CommitmentDiscountType
+
+## Example provider mappings
+
+Current column mappings found in available data sets:
+
+| Provider | Data set                 | Column                    |  Example Values  |
+|----------|--------------------------|---------------------------|------------------|
+| AWS | CUR                      | Not available   | N/A |
+| Google Cloud | BigQuery Billing Export | credit.type              | COMMITTED_USAGE_DISCOUNT, COMMITTED_USAGE_DISCOUNT_DOLLAR_BASE |
+| Microsoft | Cost Details             | Not available               | N/A |
+
+## Example usage scenarios
+
+Current values observed in billing data for various scenarios:
+
+| Provider | Data set                 | CommitmentDiscountType     | CommitmentDiscountProgram (Name TBD)     |
+|----------|--------------------------|----------------------------|------------------------------------------|
+| AWS | CUR (PurchaseOption)                   | Usage-Based                | Reserved Instances              |
+| AWS | CUR (PurchaseOption)                   | Usage-Based                | Savings Plans                   |
```

Comment:
In Savings Plans you're committing to particular spend per hour in $, shouldn't it be marked as Spend-Based?

Reply:
Thanks - this was an oversight - updated